### PR TITLE
Dependency manager

### PR DESCRIPTION
**Pull request by luoxiu**
add support for carthage and cocoapods

### CODE REVIEW

**Review by FranzBusch — DISMISSED** [review]
Awesome thanks soo much for this!

**Review by ddddxxx — CHANGES_REQUESTED** [review]
Should merge from master first since [@_exported.swift](https://github.com/cx-org/CombineX/blob/master/Sources/CombineX/%40_exported.swift) was added.

File: project.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - target: CXUtility_${platform}
+      - target: CXNamespace_${platform}
+  CXFoundation:
+    templates: [Framework]
+    dependencies:
+      - target: CXUtility_${platform}
+      - target: CXNamespace_${platform}
+      - target: CombineX_${platform}
+targetTemplates:
+  Framework:
```
Comment (ddddxxx) [inline project.yml:29]: Better setting `APPLICATION_EXTENSION_API_ONLY` to `YES`.

**Comment by jakobmygind** [comment]
Hi @luoxiu I tried this using Carthage and it builds but when installing an app using combineX to Device I get an error due to what seems to be missing info.plist:

image

File: project.yml (hunk 1 above)
Comment (luoxiu) [reply project.yml:29]: Added!

**Comment by luoxiu** [comment]
@ddddxxx done!

**Comment by luoxiu** [comment]
@jakobmygind Should fix the info.plist issue! Please try again!

**Comment by jakobmygind** [comment]
@luoxiu It works 👍

**Pull request merged**